app.js

```diff
@@ -22,4 +25,21 @@ app.use((err, req, res, next) => {
   res.status(500).json({ message: err.message })
 })
 
+
+const uriDb = 'mongodb+srv://aleksandraaleksinska:!Mu%3AjLhGccy4n2h@cluster0.oplpakl.mongodb.net/?retryWrites=true&w=majority';
```

Comment:
A to jest wyciek danych logowania (w produkcji nawet może zachaczyć o RODO/GDPR). Dane wrażliwe przechowujemy w pliku .env

Reply:
zrobione :)